scripts/astral.zs

```diff
@@ -39,3 +39,6 @@ recipes.addShaped("astralwand", <astralsorcery:itemwand>.withTag({astralsorcery:
 	[<quark:marble_speleothem>, null, null]
 ]);
 
+// Astral Starmetal via tinkers construct so it doesn't require a floating crystal 
+mods.tconstruct.Casting.addBasinRecipe(<astralsorcery:blockcustomore:1>, <ore:oreIron>, <liquid:astralsorcery.liquidstarlight>, 1000, true, 100);
+mods.astralsorcery.LightTransmutation.removeTransmutation(<astralsorcery:blockcustomore:1>, false);
```

Comment:
I think in this case the oiginal recipe adds nothing. It can be achieved later in the game but is more more difficult / impossible to automate.  
Recipes that serve no purpose should be removed.  
This helps to reduce confusion as there is a clear path to starmetal.

For the other PR I only did not remove it cause then there is no way to get the ore block which might limit players in some ways.

Reply:
Fair point!

But for **manually** farming just some starmetal, I think, gathering loads of liquid starlight is very exhausting. If the old recipe persists, you can craft a floating crystal once and then easily transmute iron ore to starmetal without that many buckets of liquid starlight.

Reply:
I agree on that point. 

It's the question of enforcing the player to quickly automate liquid starlight collection vs. providing another easy way.

I'm also unsure if the amount of liquid starlight should be reduced. I based the amount on the sf4 recipe you posted but it may need some finer balancing.